x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/set.tsx

```diff
@@ -7,8 +7,9 @@
 
 import React, { FunctionComponent } from 'react';
 import { i18n } from '@kbn/i18n';
-import { FormattedMessage } from '@kbn/i18n/react';
+import { isEmpty } from 'lodash';
```

Comment:
I believe the form lib provides an `emptyField` validator. Would that work for your use case?

Reply:
Won't work in this particular case since our validation needs to happen by checking the two fields that depend on each other and not just one field (in which case the validator you mention would've worked).